examples/client_server/README.md

```diff
@@ -0,0 +1,27 @@
+# Client Server Demo
+
+A very simple example to show peer to peer communication with rpc calls.
+
+
+## Setup
+
+1. Build the library:
+
+    `cargo build`
+
+2. Import the Godot project in the Godot editor.
+
+3. Open the `Server.tscn` scene and start the current scene with `F6`.
+
+4. Start a second instance of the Godot editor.
+
+5. In the `Remote Port` in the Debug settings of the editor (e.g. 6012).
+
+    Editor > Editor Settings > Network > Debug

```

Comment:
What does this mean? Does this setting still matter even though port numbers are hard-coded as constants in the code?

Reply:
I have added more information on this in the README. As far as I understand this is the port used to connect to the scene started from the editor for debugging.